pkg/http/requests.go

```diff
@@ -7,6 +7,7 @@ package http
 
 // OutputBindingRequest is the request object to invoke an output binding
 type OutputBindingRequest struct {
-	Metadata map[string]string `json:"metadata"`
-	Data     interface{}       `json:"data"`
+	Metadata  map[string]string `json:"metadata"`
+	Data      interface{}       `json:"data"`
+	Operation string            `json:"operation"`
```

Comment:
Can we use OperationKind here?

Reply:
This gets deserialized from the request and passed as is to the bindings function, so I would rather use a string primitive here and not force consumers of this struct to import the bindings package from contrib..